menu.py

```diff
@@ -149,64 +144,12 @@ def draw(self):
-                        pyxel.text(35, 90, "- Construction buyouts disabled", pyxel.COLOR_RED)
-                        pyxel.text(35, 120, "VIGOUR", pyxel.COLOR_ORANGE)
-                    case 6:
-                        pyxel.text(35, 70, "+ Generates 125% of usual fortune", pyxel.COLOR_GREEN)
-                        pyxel.text(35, 90, "- Generates 75% of usual wealth", pyxel.COLOR_RED)
-                        pyxel.text(35, 120, "SERENDIPITY", pyxel.COLOR_PURPLE)
-                    case 7:
-                        pyxel.text(35, 70, "+ Settlements have 200% strength", pyxel.COLOR_GREEN)
-                        pyxel.text(35, 90, "- Limited to a single settlement", pyxel.COLOR_RED)
-                        pyxel.text(35, 120, "ELIMINATION", pyxel.COLOR_RED)
-                    case 8:
-                        pyxel.text(35, 70, "+ Base satisfaction is 75", pyxel.COLOR_GREEN)
-                        pyxel.text(35, 90, "- Settlers only at level 5", pyxel.COLOR_RED)
-                        pyxel.text(35, 120, "JUBILATION", pyxel.COLOR_GREEN)
-                    case 9:
-                        pyxel.text(35, 70, "+ Units have 50% more power", pyxel.COLOR_GREEN)
-                        pyxel.text(35, 90, "- Settlements have 50% strength", pyxel.COLOR_RED)
-                        pyxel.text(35, 120, "ELIMINATION", pyxel.COLOR_RED)
-                    case 10:
-                        pyxel.text(35, 70, "+ Units have 50% more health", pyxel.COLOR_GREEN)
-                        pyxel.text(35, 90, "- Units have 75% of usual power", pyxel.COLOR_RED)
-                        pyxel.text(35, 120, "ELIMINATION", pyxel.COLOR_RED)
-                    case 11:
-                        pyxel.text(35, 70, "+ Units have 50% more stamina", pyxel.COLOR_GREEN)
-                        pyxel.text(35, 90, "- Units have 75% of usual health", pyxel.COLOR_RED)
-                        pyxel.text(35, 120, "GLUTTONY", pyxel.COLOR_GREEN)
-                    case 12:
-                        pyxel.text(35, 70, "+ Special affinity with heathens", pyxel.COLOR_GREEN)
-                        pyxel.text(35, 90, "- Always attacked by AI players", pyxel.COLOR_RED)
-                        pyxel.text(35, 120, "ELIMINATION", pyxel.COLOR_RED)
-                    case 13:
-                        pyxel.text(35, 70, "+ Thrive during the night", pyxel.COLOR_GREEN)
-                        pyxel.text(35, 90, "- Units weakened during the day", pyxel.COLOR_RED)
-                        pyxel.text(35, 120, "ELIMINATION", pyxel.COLOR_RED)
-
+                # Draw the buff and debuff text for the currently selected faction.
+                faction_detail = FACTION_DETAILS[self.faction_idx]
+                pyxel.text(35, 70, faction_detail.buff, pyxel.COLOR_GREEN)
+                pyxel.text(35, 90, faction_detail.debuff, pyxel.COLOR_RED)
+                pyxel.text(25, 170, faction_detail.rec_victory_type,
```

Comment:
The x and y coords are a bit cooked here, should be `35, 120` rather than `25, 170`.